src/pydata_sphinx_theme/theme/pydata_sphinx_theme/layout.html

```diff
@@ -52,6 +52,15 @@
 {%- endblock %}
 
 {%- block content %}
+  {# A tiny helper pixel to detect if we've scrolled #}
+  <div id="pst-scroll-pixel-helper"></div>
+
+  {# the scroll to tpo button #}
+  <button type="button" class="btn rounded-pill" id="pst-back-to-top">
+    <i class="fa-solid fa-arrow-up"></i>
+    {{ _("Scroll to top") }}
```

Comment:
I think a less ambiguous term would be helpful for example:
```suggestion
    {{ _("Back to top") }}
```

Since this is meant to be a focusable and keyboard operable element the actual action is to go back to the top rather than a mouse wheel scroll equivalent